app/views/families/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>Add a Family</p>
+<%= simple_form_for @family do |f| %>
+  <%= f.input :first_name, placeholder: 'first_name' %>
+  <%= f.input :last_name, placeholder: 'last_name' %>
+  <%= f.input :address, placeholder: 'address' %>
+  <%= f.input :animals, placeholder: 'animals' %>
+  <%= f.input :capability, placeholder: 'capability' %>
+  <%= f.input :prof, as: :radio_buttons, collection: ['Y', 'N'] %>
+  <%= f.input :preferred_gender, as: :radio_buttons, collection: ['F', 'M'] %>
+  <%= f.input :bus_driver, as: :radio_buttons, collection: ['Y', 'N'] %>
+  <%= f.input :active, as: :radio_buttons, collection: ['Y', 'N'] %>
+  <%= f.button :submit %>
+<% end %>
```

Comment:
Does Add a Family need to use translations? I haven't gotten to views yet, so I'm just wondering.

Reply:
Yes, it will need, I will once everyhting is working properly! Thanks!

Reply:
Yes, and also the field placeholders. 

For the fields labels and placeholders, check `simple_form` documentation about working with translations

Reply:
Done, thanks!